BoundingBox[] boxes
float32 image_height
float32 image_width